boefjes/boefjes/__main__.py

```diff
@@ -27,6 +27,12 @@ def cli(worker_type: str, log_level: str):
 
     queue = WorkerManager.Queue(worker_type)
     runtime = get_runtime_manager(settings, queue, log_level)
+
+    if worker_type == "boefje":
+        import boefjes.api
+
+        boefjes.api.run()
```

Comment:
I guess it will never happen that this returns without crashing, but does a return after this line make sense to ensure we don't end up starting a worker on line 36?

Reply:
This `run()` actually returns right away, because we run the API within the current Boefje runner.